spago.dhall

```diff
@@ -13,5 +13,5 @@
   , "unfoldable"
   ]
 , packages = ./packages.dhall
-, sources = [ "src/**/*.purs", "test/**/*.purs" ]
+, sources = [ "src/**/*.purs", "test/**/*.purs", "doc-test/*.purs" ]
```

Comment:
If we add code snippets to the docs directory will this pattern still pick it up (ie. it's just one flat directory) or do we need "doc-test/**/*.purs" to catch subdirectories?

Reply:
This just creates a flat directory structure, so it'll be picked up just fine. I actually want to see if there is a way to not include this is the library code, but instead pass it in as an argument to `spago` at the callsite within `purescript-doc-check-action`

Reply:
Yes, that would be much better. I saw your experiment with the —path flag and hopefully that works.